lib/sixpack.rb

```diff
@@ -2,6 +2,7 @@
 require "net/http"
 require "json"
 require "uri"
+require "securerandom"
```

Comment:
Why add secure random here?

Reply:
Hi zackkitzmiller
yeah,,its my mistake.. we are using ruby v 2.3, in order to use secure random above ruby versions 2 we need to require it explicitly. After reading some of the comments in other pull requests..came to know that sixpack is using 1.9.3 in production.
will remove this line and will push the code.
Thanks